internal/rbd/nodeserver.go

```diff
@@ -226,6 +245,27 @@ func (ns *NodeServer) stageTransaction(ctx context.Context, req *csi.NodeStageVo
 		volOptions.readOnly = true
 	}
 
+	if kernelRelease == "" {
+		// fetch the current running kernel info
+		kernelRelease, err = util.GetKernelVersion()
+		if err != nil {
+			return transaction, err
+		}
+	}
+	if !util.CheckKernelSupport(kernelRelease, deepFlattenSupport) {
+		if !skipForceFlatten {
```

Comment:
you might want to combine these two if-statements so that the code block below is not indented 2x

Reply:
Any suggestion?